README.rdoc

```diff
@@ -73,6 +73,14 @@ gitflow will automatically:
 * push the code and tags to the remote "origin"
 * and run the normal deploy task for the production stage.
 
+=== CLEANING OLD TAGS:
+Whenever you want to clean the old tags generated by this gem you can simply do
+
+    bundle exec cap <stage_name> gitflow:cleanup_tags
```

Comment:
This seems to be the only place that mentions `bundle exec`. Probably can leave it as `cap`, and do a followup PR if it should consistently use `bundle exec cap`.